### PR TITLE
homebrew-cask: update according to new sharding

### PR DESCRIPTION
**Pull request by ldennington**
Update to correctly publish casks to homebrew/homebrew-cask based on[ sharding](https://github.com/Homebrew/homebrew-cask/pull/152603).

Tested with [GCM and my fork of `Homebrew/homebrew-cask`](https://github.com/ldennington/homebrew-cask/pull/2).

### CODE REVIEW

**Review by mjcheetham — APPROVED** [review]

File: src/homebrew.ts [hunk 1]
```diff
@@ -103,7 +103,8 @@ export class Tap {
   }
 
   async getCaskAsync(name: string): Promise<Package> {
-    const filePath = `Casks/${name}.rb`;
+    const prefix = this.repo.name === 'homebrew-cask' ? 'Casks/g' : 'Casks';
```
Comment (mjcheetham) [inline src/homebrew.ts:106]:
```suggestion
    const prefix = this.repo.name === 'homebrew-cask' ? 'Casks/' + name[0] : 'Casks';
```

**Review by mjcheetham — CHANGES_REQUESTED** [review]
Actually, I just realised the sharding depends on the name of the cask. We shouldn't always be using `g`.

File: src/homebrew.ts (hunk 1 above)
Comment (mjcheetham) [reply src/homebrew.ts:106]: We also probably should always make the first character lowercase too?

Comment (ldennington) [reply src/homebrew.ts]: I think it's all fixed up now. Thanks for catching!

**Review by mjcheetham — APPROVED** [review]

**Pull request merged**